trace-C19/src/main/webapp/outputTest.html

```diff
@@ -16,7 +16,7 @@ <h2>Test Page</h2>
             </p>
         </div>
         <div class="w3-container w3-quarter w3-border w3-round w3-cyan">
-            <form class="w3-form w3-margin" method="POST" action="/outLocation">
+            <form class="w3-form w3-margin" id="zipForm" method="POST" action="/outLocation">
```

Comment:
Since you have already specified `/outLocation` and `method:post` in the fetch function, you can go ahead and remove `method="POST"` and `action="/outLocation"` from the form tag